nashcrn.Rmd

```diff
@@ -0,0 +1,277 @@
+```
+
+Replace `my_address_file_geocoder_3.2.0_score_threshold_0.5_dep_index_0.2.0_aadt_0.2.0_400m_buffer_roads_0.2.1_400m_buffer_greenspace_0.3.0_drivetime_1.1.0_cchmc.csv` with the name of the CSV file created in Step 8 and run.
+
+The **output file** is written to the same directory and in our example, will be called `my_address_file_geocoder_3.2.0_score_threshold_0.5_dep_index_0.2.0_aadt_0.2.0_400m_buffer_roads_0.2.1_400m_buffer_greenspace_0.3.0_drivetime_1.1.0_cchmc_nlcd_0.2.0_400m_buffer.csv`.
+
+**Note that the landcover container will result in mulitple rows per patient, due to multiple years of landcover data.**
+
+Example output:
+
+```{r, eval = F}
+read_csv("example_data/my_address_file_short_geocoder_3.2.0_score_threshold_0.5_dep_index_0.2.0_aadt_0.2.0_400m_buffer_roads_0.2.1_400m_buffer_greenspace_0.3.0_drivetime_1.1.0_cchmc_nlcd_0.2.0_400m_buffer.csv") %>% 
+  kable() %>% 
+  kable_styling(full_width = FALSE, bootstrap_options = "striped") %>% 
+  column_spec(35:49, bold = T, color = "white", background = "gray") %>% 
+  scroll_box(width = "900px", height = "300px")
+```
+
+More information on [landcover](https://degauss.org/nlcd/)
+
+# Step 10: Census Tract Data
+
+In this step, you will use a container to merge the output of Step 9 with a CSV file containing census tract level data. 
+
+The census tract level data is called `nash_crn_census_data_2010.csv` and should have been sent to you. Otherwise you can download the data and accompanying data dictionary [here](http://geomarker.io/NASH_CRN/).
+
+Note that any CSV opened in Microsoft Excel will not show leading zeros. If a CSV is opened in Excel then saved, the leading zeros will be truncated (e.g., `01234567891` will become `1234567891`). Avoid opening any CSVs and saving. 
+
+macOS example call: 
+
+```
+docker run --rm -v "$PWD":/tmp ghcr.io/degauss-org/census_merger:0.1.0 my_address_file_geocoder_3.2.0_score_threshold_0.5_dep_index_0.2.0_aadt_0.2.0_400m_buffer_roads_0.2.1_400m_buffer_greenspace_0.3.0_drivetime_1.1.0_cchmc_nlcd_0.2.0_400m_buffer.csv nash_crn_census_data_2010.csv
+```
+Replace `my_address_file_geocoder_3.2.0_score_threshold_0.5_dep_index_0.2.0_aadt_0.2.0_400m_buffer_roads_0.2.1_400m_buffer_greenspace_0.3.0_drivetime_1.1.0_cchmc_nlcd_0.2.0_400m_buffer.csv` with the name of the CSV file created in Step 9 and run.
+
+The **output file** is written to the same directory and in our example, will be called `my_address_file_geocoder_3.2.0_score_threshold_0.5_dep_index_0.2.0_aadt_0.2.0_400m_buffer_roads_0.2.1_400m_buffer_greenspace_0.3.0_drivetime_1.1.0_cchmc_nlcd_0.2.0_400m_buffer_nash_crn_census_data_2010_census_merger_0.1.0.csv`.
+
+
+# Step 11: Removing PHI
+
+Before sharing your data, remove the following columns from both the air pollution output file and the file created by Step 8:
+
++ `address`
++ `matched_street`
++ `matched_city`
++ `matched_zip`
++ `matched_state`
++ `lat`
++ `lon`
++ `fips_tract_id`
++ `nlcd_cell`
```

Comment:
Would it be worthwhile to put a summary of the commands that you used throughout (assuming macOS and an original input file called `my_address_file`) here at the end?  Basically a code-only synopsis of what you would run in case someone wants to copy and run that.  (Probably not any users in NASH CRN like this, but eventually could be good practice for these guides in general)

Reply:
I like this idea 👍🏼